helpers/enum.js

```diff
@@ -43,10 +43,24 @@ const dsKind = {
   DEVELOPMENT: 'Template_dev'
 };
 
+const allowedFilterValues= {
+    sort: {
+        'views': 'Most views',
+        'downloads': 'Most downloads',
+        'added_date': 'Newest',
+        'updated_date': 'Last Modified'
+    },
+    templateTypes: {
+        'tag': 'Tag',
+        'var': 'Variable'
+    },
+    categories: categories
+};
 
```

Comment:
Make sure you observe consistent spacing - all Node JS is spaced with two spaces. It's a small thing but code style and consistency :)

Reply:
Sorry I just used TAB. will check how to change that in the editor

Reply:
fixed